tools/ts_library_builder/ast_util.ts

```diff
@@ -15,7 +15,21 @@ import {
   TypeGuards,
   VariableStatement,
   VariableDeclarationKind
-} from "ts-simple-ast";
+} from "ts-morph";
```

Comment:
Are the ts_library_builder changes related to the ARM CI or just bonus improvements?

Reply:
Seems like merge cruft.

Reply:
Yet again from master.